packages/aft/lib/src/repo.dart

```diff
@@ -380,12 +381,6 @@ class Repo {
           }
           updateConstraint(package, dependent);
         }
-      } else if (type == VersionBumpType.nonBreaking) {
-        // For non-breaking changes, we still need to update all constraints
-        // since we "pin" to minor versions.
-        for (final dependent in packageDependents) {
-          updateConstraint(package, dependent);
-        }
```

Comment:
Q: If I'm following this correctly, by omitting this we will no longer see constraints on the Dart packages updated from non-major bumps. Is that correct?

Reply:
There is an example in the PR description that I think answers this question. Let me know if it doesn't.